charts/templates/ingress.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ .Values.ingress.name }}
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
+    nginx.ingress.kubernetes.io/enable-cors: "true"
+    nginx.ingress.kubernetes.io/cors-allow-origin: "https://{{ .Values.ingress.appOriginName }}"
+    nginx.ingress.kubernetes.io/cors-allow-headers: "Authorization,Referer,sec-ch-ua,sec-ch-ua-mobile,sec-ch-ua-platform,User-Agent,X-Organization,Content-Type"
+spec:
+  rules:
+    - host: {{ .Values.ingress.appOriginName }}
+      http:
+        paths:
+          - path: /
+            pathType: Prefix
+            backend:
+              service:
+                name: {{ include "growthbook.fullname" . }}
+                port: 
+                  number: {{ .Values.port.frontendPort }}
+    - host: {{ .Values.ingress.apiHostName }}
+      http:
+        paths:
+          - path: /
+            pathType: Prefix
+            backend:
+              service:
+                name: {{ include "growthbook.fullname" . }}
+                port:
+                  number: {{ .Values.port.backendPort }}
+  tls:
+    - hosts:
```

Comment:
Don't we need to handle two TLSs if appOrigin is different than apiHostname?

Reply:
I added apiHostName to tls.hosts.